src/components/SignInButton.tsx

```diff
@@ -113,18 +114,34 @@ export const SignInButton = (props: SignInButtonProps) => {
     await Metrics.captureEvent(event);
   };
 
+  const errorStreamToString = async (error: HttpError) => {
+    const body = await new Response(error.body).json();
+    return body.message || JSON.stringify(body);
```

Comment:
if there is a message return that, else return the whole error

Reply:
What if that is really long? Do we truncate the message?

Reply:
I don't believe its truncated, I think it will display the whole thing. Would it be preferable to truncate it?

Reply:
I'm not sure. This is possibly something we can just observe and address if it becomes a problem.